src/test/java/net/snowflake/client/jdbc/telemetry/TelemetryIT.java

```diff
@@ -252,9 +252,9 @@ private TelemetryClient createOAuthSessionlessTelemetry()
   // Helper function to set up and get OAuth token
   private String getOAuthToken() throws SQLException {
     Map<String, String> parameters = getConnectionParameters();
-    String testUser = parameters.get("user");
     Connection connection = getConnection();
     Statement statement = connection.createStatement();
+    statement.execute("use role accountadmin");
```

Comment:
Would this change have any implication on other tests since we're switching to use a different role? Should we consider switching back to the old role at the end of the function? Or the getConnection will create a new connection every time?

Reply:
The connection is closed in at the end of the function, so the next `getConnection` will create a new one.